bargrooves/dist/presence.js

```diff
@@ -0,0 +1,38 @@
+var presence = new Presence({
+    clientId: "641955799869947914", //The client ID of the Application created at https://discordapp.com/developers/applications
+    mediaKeys: true //Enable use and detection of media key presses
+});
+
+presence.on("UpdateData", () => {
+    //UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
+
+    //It is recommended to set up another function outside of this event function which will change variable values and do the heavy lifting if you call data from an API.
+    var presenceData = {
+        largeImageKey: "default", /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageKey: "logo-square", /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageText: "bargrooves.live", //The text which is displayed when hovering over the small image

```

Comment:
I suggest you use these images for small image keys: 
https://github.com/PreMiD/Presences/tree/master/.resources